src/main/java/com/iota/iri/service/API.java

```diff
@@ -361,7 +361,7 @@ private AbstractResponse process(final String requestString, InetSocketAddress s
             log.info("API Validation failed: " + e.getLocalizedMessage());
             return ErrorResponse.create(e.getLocalizedMessage());
         } catch (final Exception e) {
-            log.error("API Exception: ", e);
+            log.error("API Exception: " + e.getLocalizedMessage())
```

Comment:
Don't forget the semicolon ;)

https://travis-ci.org/iotaledger/iri/jobs/395274421#L534

Reply:
oops :-)